docs/running-on-mesos.md

```diff
@@ -479,6 +479,35 @@ See the [configuration page](configuration.html) for information on Spark config
 </tr>
 
 <tr>
+  <td><code>spark.mesos.driver.secret.envkey</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    If set, the contents of the secret referenced by
+    spark.mesos.driver.secret.name will be written to the provided
+    environment variable in the driver's process.
+  </td>
+  </tr>
+  <tr>
+<td><code>spark.mesos.driver.secret.filename</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    If set, the contents of the secret referenced by
+    spark.mesos.driver.secret.name will be written to the provided
+    file.  Relative paths are relative to the container's work
+    directory.  Absolute paths must already exist.  Consult the Mesos Secret
+    protobuf for more information.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.mesos.driver.secret.name</code></td>
```

Comment:
@ArtRand Looking at the protobuf definition, https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L2335, I see that unlike a `Reference`, a `message Value` does not have a name: it has a field called `data`. So, I was wondering how the user would pass in that data.
@ArtRand Looking at the protobuf definition, https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L2335, I see that unlike a `Reference`, a `message Value` does not have a name: it has a field called `data`. So, I was wondering how the user would pass in that data.